openingd/dualopend.c

```diff
@@ -596,28 +597,31 @@ static size_t psbt_input_weight(struct wally_psbt *psbt,
 				size_t in)
 {
 	size_t weight;
+	const struct wally_map_item *redeem_script = wally_map_get_integer(&psbt->inputs[in].psbt_fields, /* PSBT_IN_REDEEM_SCRIPT */ 0x04);
```

Comment:
initializer abose.

Reply:
Also, why is PSBT_IN_REDEEM_SCRIPT not exposed so we can use it here?

Reply:
@jgriffiths I may have simply missed how to access this. I (ab)use this for redeem script. witness script in a few locations.

Reply:
The blessed way to access all psbt fields is via the calls in `include/wally_psbt_members.h`, i.e. `wally_psbt_get_input_redeem_script|_len`. Unfortunately, that's kind of a sucky interface for the non-wrapped languages since its verbose and requires an alloc and copy, vs just looking up the field.

I intend to auto-generate wrappers populating a map_item for non-wrapped languages to use, but time is against me at present. Soon(tm).